vivisect/parsers/elf.py

```diff
@@ -582,10 +583,6 @@ def applyRelocs(elf, vw, addbase=False, baseaddr=0):
                         logger.info(r.tree())
                        
                 else:
-                    symidx = Elf.getRelocSymTabIndex(r.r_info)
-                    sym = elf.getDynSymbol(symidx)
```

Comment:
here's the actual bug which was crashing

Reply:
Do you know how much this affects non-static binaries, or if we've already dropped into here (because name is None), we're guaranteed to be in static town (for this reloc)?

Reply:
so... if you look through the rest of the function, neither of these is used again.